src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -1,6 +1,6 @@
 package edu.kis.vh.nursery;
 
-public class defaultCountingOutRhymer {
+public class DefaultCountingOutRhymer {
 
 	private int[] NUMBERS = new int[12];
```

Comment:
Array is not a constant (lowercase).

Lack of collaborator work. Extracting constants is done incorrectly. Hints (1.4.w7): tiny.cc/998muz ?